duckduckgo/iOS
# fix content blocking in detection code

Task/Issue URL: https://app.asana.com/0/414235014887631/628429840777099
Tech Design URL:
CC:

**Description**:

Ensure that dynamic Image.src resources are blocked and that XHRs are blocked and cleaned up correctly by calling abort/apply as appropriate.

**Steps to test this PR**:

1. Visit cnn.com
1. In the Xcode console search for "type = image" and find an image which is reported as blocked
1. Using Safari debug console check the Network tab to ensure that URL does not appear
1. Repeat the above using "type = xmlhttprequest"

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
